recipes/pyomero-upload/meta.yaml

```diff
@@ -6,11 +6,14 @@ package:
 
 source:
   url: https://github.com/SynthSys/pyOmeroUpload/archive/v1.0.0-rc0.tar.gz
-  sha256: 80b983792b8f54caa86b9815806be9320a5eab358f84d94fef1e3ca2a42981dc
+  sha256: 9bdb948b9720bb4d0fc761be19aa9c05a062195e06e74e0339d0d4fd6e3b3dd7
+  # git_url: https://github.com/SynthSys/pyOmeroUpload.git
+  # git_rev: {{ PYOMERO_VERSION }}
+  # md5: unused
 
 build:
   noarch: python
-  number: 0
+  number: 1 
 
 # We are including a pinned openssl requirement because the ZeroC packages need to be re-compiled
```

Comment:
This should re recompiled, can you try remoing it

Reply:
Removed from latest commit